app/lib/exchange-rate/providers/coinbase.ts

```diff
@@ -26,7 +27,7 @@ export class Coinbase implements ExchangeRateProvider {
 
     const data = await response.json();
     return {
-      'BTC-USD': Number.parseFloat(data.data.amount),
+      'BTC-USD': new Big(data.data.amount).toString(),
```

Comment:
if I am not wrong, you are doing conversion from string to big and then back to string. I think we can just do `'BTC-USD': data.data.amount`